src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeData(readFrom(fromFileName), toFileName);
```

Comment:
```suggestion
        String report = readFrom(fromFileName);
        writeData(report, toFileName);
```
я б супер сильно не захоплювався інлайном для читаймості)